integration/tests/basic/helpers/assert_message.py

```diff
@@ -4,6 +4,7 @@
 class ErrorMessage(Enum):
     NEGATIVE_VALUE = "Resulting wei value must be between 1 and "
     INSUFFICIENT_FUNDS = "insufficient funds for transfer"
```

Comment:
this too errors exist? do you difference?

Reply:
we get INSUFFICIENT_FUNDS error when there are not enough neons on account to deploy a contract, for example
and INSUFFICIENT_BALANCE error when value in transaction is bigger then account balance